### ISSUE TITLE
Make it possible to compute area under the curve

### ISSUE THREAD

**Issue by samreid** [phetsims/blackbody-spectrum#7]
I received this request from a Physics teacher

> I have a PhET request. The Blackbody simulation is a great way to introduce my Modern Physics kids to the world of quantum mechanics. I do a little historical talk at the beginning to get them to understand why quantum mechanics was developed. As part of the activity that I have them do, I have them use the simulation to take measurements and confirm some of the mathematical relationships.
> 
> One thing that would be cool: If I could have them find the area under the blackbody curve at different temperatures, then they could compare total emittance to temperature, which I believe is the Raleigh Jeans Law. It doesn't have to be a full integral function (although that would be sweet!). Even if there was a grid that could be enabled for them to estimate the area, that would be great.
> 
> Thanks for everything!

**Comment by samreid** [comment]
@ariel-phet can you please review/delegate/prioritize and assign someone to write a response?  I'm not familiar with the physics for this situation.

**Comment by ariel-phet** [comment]
@samreid please forward me the email, I will respond (note editing your initial comment to remove some info)

**Comment by ariel-phet** [comment]
@samreid once you have forwarded the email, please clear assignee for this issue

**Comment by samreid** [comment]
@ariel-phet I forwarded you the email, unassigning.

**Comment by DianaTavares** [comment]
Proposal
![screen shot 2018-01-30 at 2 47 11 pm](https://user-images.githubusercontent.com/26031501/35593329-e1abd0d6-05cc-11e8-8f53-506946ea5559.png)

**Comment by andrealin** [comment]
In master, area under the curve has been computed and displayed according to the mockup [above.](https://github.com/phetsims/blackbody-spectrum/issues/7#issuecomment-361746293)

There was a discussion about where to put the label in https://github.com/phetsims/blackbody-spectrum/issues/25, but I believe this issue here can be closed. Assigning to @jbphet to review and close.

**Comment by jbphet** [comment]
@samreid - do you have the address of the teacher who sent you the request?  Don't put it in your response (since this is public), but if you do have the address, I'd like to leave this issue open and deferred until publication, and which point it would be nice to follow up and let the teacher know that the feature exists.

**Comment by samreid** [comment]
Yep, I can still reach out to the teacher who suggested this.

@SaurabhTotey for future commits, please reference the issue in the commit message, so the work shows up in the thread.  For example, see: https://github.com/phetsims/area-model-common/issues/130

**Comment by jbphet** [comment]
Added label 'fixed-awaiting-deploy', which is perhaps a bit unusual for an unpublished sim, but I want to leave this open and unassigned once all work on it is complete so that we remember to let the teacher know once the HTML5 sim with this feature is published.

**Comment by ariel-phet** [comment]
@arnabp please check in with @samreid once the sim is published as noted above

**Comment by arnabp** [comment]
@samreid Sim is published, you can now follow up with the teacher with the good news on the new feature!

**Comment by samreid** [comment]
I responded to the teacher, thanks everyone for your work on this issue.  Closing.
